src/store/Store.js

```diff
@@ -47,7 +53,7 @@ export default class Store {
 		});
 
 		window.addEventListener('visibilitychange', () => {
-			!this._state.minimized && !this._state.triggered && parentCall('openWidget');
+			!this._state.minimized && parentCall('openWidget');
```

Comment:
Sorry I didn't get this change. Why are we not checking for the `triggered` property now?